dashboard/src/features/shared/components/BaseList/EmptyList.jsx

```diff
@@ -13,28 +13,17 @@ class EmptyList extends React.Component {
     if (!this.props.loadedOnce) {
       emptyBlock = <span>LOADING…</span>
     } else if (this.props.showFirstTimeFlow && this.props.skipCreate) {
-      emptyBlock = <div>
-        <span className={styles.emptyLabel}>
-          There are no {this.props.objectName.toLowerCase()}s on the blockchain
-        </span>
-      </div>
+      emptyBlock = <div>{this.props.emptyContent}</div>
     } else if (this.props.showFirstTimeFlow) {
       emptyBlock = <div>
-        <span className={styles.emptyLabel}>Create your first {this.props.objectName}</span>
+        {this.props.emptyContent}
```

Comment:
I think we should come up with a better prop name for this, or make `emptyContent` and object so that it's rendered as:

```
{this.props.emptyContent.firstTime}
```